lib/structuraid_core/engineering/analysis/footing/centric_combined_two_columns.rb

```diff
@@ -0,0 +1,85 @@
+module StructuraidCore
+  module Engineering
+    module Analysis
+      module Footing
+        class CentricCombinedTwoColumns < Base
+          include Utils::Data
+          include Utils::Centroid
+          include Utils::ShearMoment
+
+          def initialize(footing:, loads_from_columns:, section_direction:)
+            if ORTHOGONALITIES.none?(section_direction)
+              raise Engineering::Analysis::SectionDirectionError.new(section_direction, ORTHOGONALITIES)
+            end
+
+            @footing = footing
+            @loads_from_columns = loads_from_columns
+            @section_direction = section_direction
+          end
+
+          def build_geometry(coordinates_system)
+            @coordinates_system = coordinates_system
+
+            coordinates_system.relative_locations.clear
+            relativize_loads
+
+            aligner_vector = coordinates_system.relative_locations.first.to_vector
```

Comment:
what is the idea behind using `relative_locations.first` here? Is it safe to depend on the fact that the first relative location is the one we want to use to align the coordinates system?

Reply:
![Mi%20primer%20FlipaClip](https://user-images.githubusercontent.com/26731448/228654741-c28e280c-9492-4df3-af98-6621afd9d408.gif)


Once the coordinates system is aligned with the aligner_vector, the task is to add edges to the relative locations stored at the coordinates_system. Take a look at the gif.

A change: now the alignment is working with the last of the load's location

Reply:
This is with the aim of organizing `relative_locations` to enable the ability of compute stretch between `edge_1`-`load_1`, `load_1`-`load_2` and `load_2 `-`edge_2`

Reply:
got it. This has to go to notion somewhere!